static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts

```diff
@@ -142,36 +138,37 @@ const addContentClass = (adSlotNode: HTMLElement) => {
 };
 
 /**
- * Avoid CLS when an advert is refreshed, by setting the
+ * Prevent CLS when an advert is refreshed, by setting the
  * min-height of the ad slot to the height of the ad.
- *
- * Doesn't work for fluid ads, because we don't know the height of a fluid ad at this point.
  */
-const setMinHeightOfAdSlot = (advert: Advert): void => {
-	if (
-		advert.id !== 'dfp-ad--top-above-nav' ||
-		advert.size === null ||
-		advert.size === 'fluid'
-	) {
+const setAdSlotMinHeight = (advert: Advert): void => {
```

Comment:
Any thoughts on where this should live? It's now only being called by the file `on-slot-viewable.ts`. I left it here as I felt it made more sense to be in the `render-advert.ts`, but not sure whether that's worth it or not, or even to create a new file for it.

Reply:
If its only used by `on-slot-viewable.ts` then perhaps that it should live there?